target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -227,6 +227,12 @@ mikrotik,rbm11g)
 miniembplug)
 	set_wifi_led "$boardname:red:wlan"
 	;;
+xiaomi,mir3p)
+	ucidef_set_led_switch "wan-amber"  "WAN (amber)"  "$boardname:amber:wan"  "switch0" "0x10" "0x08"
+	ucidef_set_led_switch "lan1-amber" "LAN1 (amber)" "$boardname:amber:lan1" "switch0" "0x02"  "0x08"
```

Comment:
There's a unnecessary whitespace between 「"0x02"  "0x08"」

Reply:
@EXSERENS yeah sorry. fixed that.